sympy/polys/polytools.py

```diff
@@ -6218,8 +6218,20 @@ def sqf_list(f, *gens, **args):
     (2, [(x + 1, 2), (x + 2, 3)])
 
     """
-    return _generic_factor_list(f, gens, args, method='sqf')
-
+    options.allowed_flags(args, ['frac', 'polys'])
+    opt = options.build_options(gens, args)
+    f = sympify(f)
+    if isinstance(f, Expr) and not f.is_Relational:
```

Comment:
`Relational` is no more a subclass of `Expr` so the condition `f.is_Relational` is not needed.

I would also write

    if not isinstance(f, Expr):
        raise PolynomialError("a polynomial expression expected, got %s" % f)

Then no extra indentation is necessary for the next lines.

Reply:
This test may not be needed if `_poly_from_expr` is used. It will also sympify the argument.